src/github_runner_image_builder/cli.py

```diff
@@ -115,3 +115,89 @@ def run(  # pylint: disable=too-many-arguments
     if callback_script:
         # The callback script is a user trusted script.
         subprocess.check_call([str(callback_script), image_id])  # nosec: B603
+
+
+@main.command(name="init_external")
+@click.argument("cloud_name")
+def initialize_external(cloud_name: str) -> None:
+    """Initialize builder CLI function wrapper.
+
+    Args:
+        cloud_name: The cloud to use from the clouds.yaml file. The CLI looks for clouds.yaml in
+            paths of the following order: current directory, ~/.config/openstack, /etc/openstack.
+    """
+    arch = get_supported_arch()
+    openstack_builder.initialize(arch=arch, cloud_name=cloud_name)
+
+
+@main.command(name="experimental-run-external")
+@click.argument("cloud_name")
+@click.argument("image_name")
+@click.argument("flavor")
+@click.argument("network")
```

Comment:
See if you have any ideas for how we could not have these

Reply:
E.g., just see which flavours are available and pick one? Perhaps also for network?

Reply:
Changed to optional flag. If not provided it will try best effort to determine them both.